src/commands/dev/mod.rs

```diff
@@ -47,12 +49,19 @@ pub async fn dev(
         let server_config = server_config.clone();
         async move {
             Ok::<_, failure::Error>(service_fn(move |req| {
-                preview_request(
-                    req,
-                    client.to_owned(),
-                    preview_id.to_owned(),
-                    server_config.clone(),
-                )
+                let client = client.to_owned();
+                let preview_id = preview_id.to_owned();
+                let server_config = server_config.clone();
```

Comment:
do we need to clone this twice? (once in line 49, once here?) I feel like we can put variables like client, preview_id, and server_config in the arguments section `move |args|` and avoid unnecessary clones :)